AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -245,7 +245,7 @@ - (void)dealloc
 
 - (UIResponder *)nextResponder
 {
-  return self.view.superview;
+  return [self.view nextResponder];
```

Comment:
Woah.  This is almost as bad as the "not handling a touch? let's forward it up the view hierarchy instead of the class inheritance heirarchy!" bug.  Nice find.

Have you confirmed that we're not depending on the old (wrong) behaviour anywhere?